Automattic/wp-calypso
# Build: use shonkwrap to generate npm-shrinkwrap.json and update docs

This PR adds a newly generated npm-shrinkwrap.json from [shronkwrap](https://github.com/skybet/shonkwrap). The resolved/from fields have been removed where possible to allow us to use local repository settings.

## Testing Instructions 
- Run `make distclean` to delete local node_modules
- Run `npm install`
- Verify that Calypso works as expected and that tests pass.

cc @rralian @aduth @blowery
